lib/error-matcher.js

```diff
@@ -23,12 +23,21 @@ module.exports = (function () {
   util.inherits(ErrorMatcher, EventEmitter);
 
   ErrorMatcher.prototype._goto = function () {
-    var file = this.cwd + path.sep + this.currentMatch[0].file;
-    if (!this.currentMatch[0].file) {
+    var file = this.currentMatch[0].file;
+    if (file) {
+      var oldDir = process.cwd();
+      process.chdir(this.cwd);
```

Comment:
Well no. But I tend to refrain from doing stuff which change global behavior. If a user uses any other package which depends on current working directories, it's unlikely that package will work well with build.

Here's another solution. We could use [fs.isAbsolute](https://nodejs.org/api/path.html#path_path_isabsolute_path) to check if path is absolute. This takes into account all different platforms node is supported on . If the path is absolute then we just use `path` otherwise we use `this.cwd + path.sep + path`. What do you think?